safe_transaction_service/history/serializers.py

```diff
@@ -234,7 +235,14 @@ def validate(self, data):
 
     def save(self, **kwargs):
         safe_tx_hash = self.validated_data["contract_transaction_hash"]
-        origin = self.validated_data["origin"]
+        # Origin field on db is a JsonField
+        if self.validated_data["origin"]:
```

Comment:
I think you should do this on `def validate_origin` and already return the object (or string) there

Reply:
I will include both changes in different PR.